clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -35,6 +36,11 @@ sealed abstract class QueryCommand[CI <: ClioIndex](val index: CI)
   def includeDeleted: Boolean
 }
 
+sealed abstract class RawQueryCommand[CI <: ClioIndex](val index: CI)
+    extends RetrieveAndPrintCommand {
+  def queryInput: File
```

Comment:
Would name this `queryInputPath` or something else to distinguish it from the non-raw `QueryInput`

Reply:
Ooo or! Something like this type-fu:
```scala
sealed abstract class QueryCommand[CI <: ClioIndex, QI](val index: CI) extends RetrieveAndPrintCommand {
  def queryInput: QI
}
sealed abstract class RawQueryCommand[CI <: ClioIndex](val index: CI) extends QueryCommand[CI, File]
sealed abstract class SimpleQueryCommand[CI <: ClioIndex](val index: CI) extends QueryCommand[CI, index.QueryInputType]
```
And then the executor can abstract over the type of input.